compose/material3/material3-xr/api/current.txt

```diff
@@ -0,0 +1 @@
+// Signature format: 4.0
```

Comment:
As `material3-xr`/`material3-xr-adaptive` are not a part of the Material3 version group, we need to exclude them from the PR and [the script](https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/scripts/copy/copyComposeMaterial3.sh), and keep it only in `integration`.